docs/source/getting_started.rst

```diff
@@ -88,3 +88,7 @@ Probabilities: {
 }
 
 Now that you can run some code, let's continue with a tutorial to learn Perceval syntax.
+
+You can also check out the Quandela Training Centre_ for more Lessons and Tutorials.
```

Comment:
Hello, I'm not sure this is the right place to put this link.
I think it would be more suited in the index.rst file (perhaps in the `related projects` or after the link to `Welcoming Contributors`)